### ISSUE TITLE
Allow to lock a command without executing it

### ISSUE THREAD

**Issue by gggeek** [kaliop/kaliop-lock-command-bundle#5]
It would be nice to be able to preventively lock a command, eg

`php app/console kaliop:command-lock:lock mycommand:that:does_stuff`

**Comment by bOnepain** [comment]
Indeed, it could be a nice feature. Ready to implement it @gggeek ?
